docs/galapagos_user_guide.md

```diff
@@ -0,0 +1,460 @@
+   To do this, go back to the certificate dialog for the respective environment and switch to
+   the "Extend existing Certificate" tab:
+   
+   
+   ![](galapagos_user_guide_images/extend-cert.png)
+
+   Note that the expiration date is also shown in this dialog. You can always check the expiration date here, but 
+   you will also be notified by Galapagos when the certificate is about to expire.
+
+   Here you will find an openssl command as well. However, this requires the private key for the existing certificate. If you "only" have the .p12 file, you can
+   extract the private key as follows: 
+   
+```
+winpty openssl pkcs12 -in <myclientkeystore.p12> -nocerts -nodes -out <myprivatekey.key>
+```
+
+   You will then be asked for the password for your P12 keystore on the command line, so don't forget
+   the winpty (under Windows), otherwise the openssl process will hang! 
+   
+   Otherwise you can proceed as described above. Execute the command that Galapagos lists for you and
+   copy the CSR to Galapagos into the text box. You will now receive a certificate that contains the
+   same identifier as your previous certificate (and uses the same private key), but is (depending on
+   the environment) valid for 1 or 2 years. The previous certificate can still be used (as long as it
+   is valid), just exchange it at the next opportunity.
+   
+### Replace certificate
+
+   If you have lost the private key or have to recreate the client certificate for any other reasons, you can proceed in exactly the same way as when re-issuing the 
+   certificate (but not as with the renewal). Please note that the previous certificate immediately
+   loses its validity when the new certificate is generated! If the application is already running 
+   in a prod-environment with the private key and you "just" can no longer access it, you could kill your 
+   application by generating a new certificate! Galapagos will warn you when trying to generate a new certificate, if you already 
+   have one:
+   
+   ![](galapagos_user_guide_images/cert-warn.png)
+   
+## Create developer certificate   
+       
+   Depending on your team organization and deployment strategy, not every developer should have the private key of 
+   your application - especially not for production. You may still have to be able to look 
+   at the topics during production, for example. There are tools for this, but of course these also require a client
+   certificate (and the trust store) for the connection to connect to Kafka. 
+   
+   Therefore, you can get a developer certificate with Galapagos. This basically grants you the same rights as the
+   applications of which you are the Topic Administrator ("My Applications"), but are always very limited in time.  
+   Currently the times are 90 days on DEV / 
+   , 30 days on INT and 10 days on PROD. Then you have to create a new
```

Comment:
This is Hermes specific information and can be configured by Galapagos admins, so better remove here.